src/main/java/org/launchcode/liftoffrecipetracker/controllers/RecipeController.java

```diff
@@ -0,0 +1,58 @@
+package org.launchcode.liftoffrecipetracker.controllers;
+
+import org.launchcode.liftoffrecipetracker.models.Recipe;
+import org.launchcode.liftoffrecipetracker.models.data.RecipeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+
+@Controller
+@RequestMapping("recipes")
+public class RecipeController {
+
+	@Autowired
+	private RecipeRepository recipeRepository;
+
+	@GetMapping
+	public String displayAllRecipes(Model model) {
+		model.addAttribute("title", "All Recipes");
+		model.addAttribute("recipes", recipeRepository.findAll());
+		return "index";
```

Comment:
This returns the root index.html which would be our homepage. Views in this controller should all follow the format you have in your subsequent methods "recipes/file_name."